src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  display: flex;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+  width: 100%;
+}
+
+.header-content {
+  text-decoration: none;
+  display: flex;
+  position: relative;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  gap: 20px;
+}
+
+li {
+  display: inline-block;
+  margin-right: 20px;
+}
```

Comment:
<img width="866" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/de2689f5-225b-469a-815e-df78446b5676">
better to give the `nav__item` name